src/compat/should_use_webkit_media_keys.ts

```diff
@@ -15,11 +15,12 @@
  */
 
 import { isSafari } from "./browser_detection";
+import { WebKitMediaKeysConstructor } from "./eme/custom_media_keys/webkit_media_keys_constructor";
 
 // On Safari 12.1, it seems that since fairplay CDM implementation
 // within the browser is not standard with EME w3c current spec, the
 // requestMediaKeySystemAccess API doesn't resolve positively, even
 // if the drm (fairplay in most cases) is supported.
 export default function shouldUseWebKitMediaKeys() {
```

Comment:
Now I don't understand this function anymore and how it is used and needed.
It seems now to be only used to:
  1. not rely on the "encrypted" event
  2. not rely on `navigator.requestMediaKeySystemAccess`.

It doesn't immediately click with me why a function called that way would do both. Reading all the code is not clear either